v2/concordium/service.proto

```diff
@@ -53,4 +53,24 @@ service Queries {
   // Get information about the current state of consensus.
   rpc GetConsensusInfo (Empty) returns (ConsensusInfo);
 
+  // Get the cryptographic parameters in a given block.
+  rpc GetCryptographicParameters (BlockHashInput) returns (CryptographicParameters);
+
+  // Get information, such as height, timings, and transaction counts for the given block.
+  rpc GetBlockInfo (BlockHashInput) returns (BlockInfo);
+
+  // Get all the bakers at the end of the given block.
+  rpc GetBakerList (BlockHashInput) returns (stream BakerId);
+
+  // Get status information about a given pool.
+  rpc GetPoolStatus (PoolStatusRequest) returns (PoolStatus);
+
+  // Get status information about the passive delegators.
+  rpc GetPassiveDelegationStatus (BlockHashInput) returns (PassiveDelegationStatus);
+
+  // Get a stream of live blocks at a given height.
+  rpc GetBlocksAtHeight (BlocksAtHeightRequest) returns (stream BlockHash);
```

Comment:
I would recommend returning a list here and not streaming.

In most cases this will be one or two blocks. And the API is simpler like that.